tasks/window/compute_do.go

```diff
@@ -190,32 +191,49 @@ func (t *WdPostTask) DoPartition(ctx context.Context, ts *types.TipSet, maddr ad
 				return nil, xerrors.Errorf("post generation randomness was different from random beacon")
 			}
 
+			for _, skippedNum := range computeSkipped {
```

Comment:
an array that we just keep overwriting this one value from? Should this be a sum?
More comments please.